### DISCUSSION TITLE
New Media Panel Nest Logo ?????

### DISCUSSION THREAD

**Discussion by thorrrr** [home-assistant/frontend#11928] [Other feature requests]
HI Guys
Loving the new media one slight caveat i see a Nest Panel in mine which is strange as i only have a thermostat! 🙂 Can it  be configured not to add or a way to remove it as it is pointless 

See [here ](https://xbackbone.daddycloud.co.uk/Piqa4/pArOToco60.jpg)& [here ](https://xbackbone.daddycloud.co.uk/Piqa4/paVEyiYE05.jpg)

**Comment by spacegaier** [comment]
Currently you can't and ideally you shouldn't have to. It might be worth raising that with the Nest integration since it is up to the integration to tell HA if there are usable media sources (and apparently currently it does create those).

The frontend merely shows what the backend integration tells it to.
